modules/persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure_{context}"]
+= Pre-provisioning across subscriptions for Azure File by creating a PV and PVC:
+
+.Prerequisites
+* Installed Azure cluster with the service principal or managed identity as Azure identity
+
+* Access to another subscription in the tenant with the cluster  
+
+* Logged in to the Azure CLI
+
+.Procedure
+. For the desired existing Azure File share, record the resource group, storage account, storage account key, and Azure File name. These values are used for the next steps.
```

Comment:
Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#d), do not use _desire_, use _want_. However, I don't like _wanted_ here. I see _your_ earlier in the preview. Can you come up with a better word/rewrite? 
```suggestion
. For your Azure File share, record the resource group, storage account, storage account key, and Azure File name. These values are used for the next steps.
```